src/components/ChangePlan/Card.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import classNames from 'classnames';
+
+export const Card = ({ children, className, highlighted, ...rest }) => {
+  return (
+    <div className={classNames({ 'dp-card': true, 'dp-highlighthed': !!highlighted })} {...rest}>
```

Comment:
Falta `className`, no?

```suggestion
    <div className={classNames({ 'dp-card': true, 'dp-highlighthed': !!highlighted }, className)} {...rest}>
```

Reply:
fixed